browser/net/url_context.h

```diff
@@ -101,6 +101,12 @@ struct BraveRequestInfo {
   GURL ipfs_gateway_url;
   bool ipfs_auto_fallback = false;
 
+  // Used to keep track of state between a primary adblock engine query and one
+  // after CNAME uncloaking the request.
+  bool did_match_rule = false;
```

Comment:
let's not add new members here when not strictly necessary. Moreover, reading/writing them on different sequences is not actually correct and at some point we will refactor this structure.

Reply:
Am I right that you only need `did_match_important`, to check in `OnShouldBlockRequestResult` ? And two others can be left alone?

Reply:
nope, the two others are checked/populated inside of `adblock-rust`

Reply:
do you mean they should persist between the initial check and the uncloaked check?

Reply:
correct

Reply:
fixed